.ci/scripts/run-gradle.sh

```diff
@@ -41,4 +41,4 @@ else
 fi
 
 set -e
-$GRADLEW -S --max-workers=$MAX_WORKERS $@
+$GRADLEW -S --max-workers=$MAX_WORKERS "$@"
```

Comment:
Please let me know if additional testing (besides main pipeline) is required for this change

Reply:
Seems reasonable to me. Have you tested running the pipeline that caused this issue to verify it fixes your problem?

Reply:
LGTM. have you manually triggered the pipeline that failed from the patch branch to verify the fix works here too?